### PR TITLE
Add sample skeletons

### PR DESCRIPTION
**Pull request by mmarionncern**
On top of  #207 , contains the skeleton of the sample files copied from 2017, as well as the findSample script that looks into the DB to create a new sample file with new samples inside

### CODE REVIEW

**Comment by gpetruc** [comment]
Why do we need a new samples file for the same RunIISummer16 MiniAOD v2 processing?

File: RootTools/python/samples/samples_13TeV_RunIISummer16Legacy.py [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+kreator = ComponentCreator()
+
+### NB: Commented lines refer to samples available in RunIIFall15MiniAODv2 but not yet in RunIISpring16MiniAODv1
+
+### ----------------------------- 25 ns ----------------------------------------
+
+# TTbar cross section: NNLO, https://twiki.cern.ch/twiki/bin/view/LHCPhysics/TtbarNNLO (172.5)
+TTJets = kreator.makeMCComponent("TTJets", "/TTJets_TuneCUETP8M2T4_13TeV-amcatnloFXFX-pythia8/RunIISummer16MiniAODv2-PUMoriond17_backup_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM", "CMS", ".*root", 831.76)
+TTJets_ext = kreator.makeMCComponent("TTJets_ext", "/TTJets_13TeV-amcatnloFXFX-pythia8/RunIIFall15MiniAODv2-PU25nsData2015v1_76X_mcRun2_asymptotic_v12_ext1-v1/MINIAODSIM", "CMS", ".*root", 831.76)
+TT_pow = kreator.makeMCComponent("TTbar", "/TT_TuneCUETP8M1_13TeV-powheg-pythia8//MINIAODSIM", "CMS", ".*root", 831.762)
```
Comment (gpetruc) [inline RootTools/python/samples/samples_13TeV_RunIISummer16Legacy.py]: missing part of the dataset name

File: RootTools/python/samples/samples_13TeV_RunIISummer16Legacy.py [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+ZGTo2LG = kreator.makeMCComponent("ZGTo2LG", "/ZGTo2LG_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6_ext1-v1/MINIAODSIM", "CMS", ".*root", 131.3)
+WW2L2NuDouble = kreator.makeMCComponent("WW2L2NuDouble", "/WWTo2L2Nu_DoubleScattering_13TeV-pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM", "CMS", ".*root", 1.64*(3*0.108)*(3*.108))
+WpWpJJ = kreator.makeMCComponent("WpWpJJ", "/WpWpJJ_EWK-QCD_TuneCUETP8M1_13TeV-madgraph-pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM", "CMS", ".*root", 0.03711)
+
+DiBosons = [
+WWTo2L2Nu,
+WWTo1L1Nu2Q,
+ZZTo2L2Nu,
+ZZTo4L,
+WZTo1L3Nu,
```
Comment (gpetruc) [inline RootTools/python/samples/samples_13TeV_RunIISummer16Legacy.py]: this is not defined

File: RootTools/python/samples/samples_13TeV_RunIISummer16Legacy.py [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+TTTT_ext = kreator.makeMCComponent("TTTT_ext", "/TTTT_TuneCUETP8M2T4_13TeV-amcatnlo-pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM", "CMS", ".*root", 0.009103)
+
+Rares = [
+TTTT,
+TTTT_ext,
+]
+
+### ----------------------------- summary ----------------------------------------
+
+mcSamples = TTs + SingleTop + VJets + DYJetsM50HT + DYJetsM5to50HT + WJetsToLNuHT + WNJets + GJetsHT + ZJetsToNuNuHT + QCDHT + QCDPtbcToE + QCDPt + QCDPtEMEnriched + [QCD_Mu15] + QCD_Mu5 +  DiBosons + TriBosons + TTV + Higgs + Rares
```
Comment (gpetruc) [inline RootTools/python/samples/samples_13TeV_RunIISummer16Legacy.py]: `DYJetsM50HT` , `DYJetsM5to50HT` ,`WJetsToLNuHT`, `WNJets`, `GJetsHT`, `ZJetsToNuNuHT`, `QCDHT`, `QCDPt` not defined

**Review by gpetruc — CHANGES_REQUESTED** [review]
also samples_13TeV_RunIISummer16Legacy_ExtraSamples.py is buggy (just loading it with python shows that there are undefined samples)

**Comment by mmarionncern** [comment]
@gpetruc This is rather a template that will be updated once we have the samples rather than a new sample file for RunIISummer16. I can change the name to make more explicit if you wish

**Comment by gpetruc** [comment]
Uhm, it's still not clear to me what's the goal.
For Fall17, we have a sample file that is being populated with new samples.
For the legacy 2016, I would wait until the legacy re-miniAOD is ongoing and first samples arrive before adding a new sample file.
So, I would prefer to close this now, and re-open it when it will be needed.

**Comment by mmarionncern** [comment]
No problem for me, I think it was wanted by @peruzzim, but maybe I misunderstood. If both of you agree, I close this one and wait till we have the full Fall17 picture

**Pull request closed without merging**